src/models/models.js

```diff
@@ -133,8 +80,6 @@ const OrderSchema = new Schema({
   OrderNumber: Number,
   Date,
```

Comment:
Should we specify the type of this attribute?

Reply:
Yeap, sorry! Let me fix it! Thanks👌
I'll set it as a string